src/utils/popups.js

```diff
@@ -141,7 +149,7 @@ const sitePopupHTML = (title, displayProps, url) => {
   const associateWCLink = `<button class="govuk-button btn-sm mt-3"
                                    id="associate-wc-link-button"
                                    style="font-size:0.9rem"
-                                   data-site-uri="${displayProps.URI}"
+                                   data-site-uri="${displayProps.URI || url}"
```

Comment:
Same as line 376 ⬇️ this is a workaround for allowing the Bristol sites to have their nearest WClinks highlighted even though they don't have an EA-like URI